doc/man7/EVP_PKEY-ML-DSA.pod

```diff
@@ -0,0 +1,114 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY-ML-DSA, EVP_KEYMGMT-ML-DSA,
+EVP_PKEY-ML-DSA-44, EVP_PKEY-ML-DSA-65, EVP_PKEY-ML-DSA-87
+- EVP_PKEY ML-DSA keytype and algorithm support
+
+=head1 DESCRIPTION
+
+ML-DSA implements the algorithms B<ML-DSA-44>, B<ML-DSA-65> and B<ML-DSA-87>. 
+The key types B<EVP_PKEY_ML_DSA_44>, B<EVP_PKEY_ML_DSA_65> and
+B<EVP_PKEY_ML_DSA_87> are implemented in OpenSSL's default provider.
+These implementations support the associated key, containing the public key I<pub>
+and the private key I<priv>.
+
+Each of the different key types has an associated security category.
+This value is one of 2, 3 or 5 for key types B<ML-DSA-44>, B<ML-DSA-65>
+and B<ML-DSA-87> respectively, which correspond to security strengths of
+128, 192 and 256 repsectively.
+
+=head2 Keygen Parameters
+
+=over 4
+
+=item "seed" (B<OSSL_PKEY_PARAM_ML_DSA_SEED>) <octet string>
+
+The seed used to generate the private and public key components in a
+deterministic manner. This should only be used for testing purposes.
+The length of the value supplied must be 32 bytes.
+When this value is not supplied the seed is generated randomly using a DRBG. 
+
+=item "properties" (B<OSSL_PKEY_PARAM_PROPERTIES>) <utf8_string>
+
+Sets properties to be used when fetching algorithm implementations used for
+ML-DSA hashing operations.
+
+=back
+
+Use EVP_PKEY_CTX_set_params() after calling EVP_PKEY_keygen_init().
+
+=head2 Common ML-DSA parameters
+
+In addition to the common parameters that all keytypes should support (see
+L<provider-keymgmt(7)/Common parameters>), the implementation of these key types
```

Comment:
That `Common parameters` isn't written that way in that document, it is rather: `Common Information Parameters`.